BDArmory/BDArmory.csproj

```diff
@@ -279,6 +279,15 @@
     <Compile Include="Evolution\EngineGimbalAxisNudgeMutation.cs" />
     <Compile Include="Evolution\ControlSurfaceNudgeMutation.cs" />
     <Compile Include="Evolution\EngineGimbalNudgeMutation.cs" />
+    <Compile Include="Competition\OrchestrationStrategy.cs" />
+    <Compile Include="Competition\RankedFreeForAllStrategy.cs" />
+    <Compile Include="Competition\WaypointFollowingStrategy.cs" />
+    <Compile Include="Competition\SpawnStrategy.cs" />
+    <Compile Include="Competition\CircularSpawnStrategy.cs" />
+    <Compile Include="Competition\PointSpawnStrategy.cs" />
+    <Compile Include="Competition\TournamentCoordinator.cs" />
+    <Compile Include="Competition\VesselSource.cs" />
+    <Compile Include="Competition\ListSpawnStrategy.cs" />
```

Comment:
I think for organisation, the various spawn strategies ought to be placed in a subfolder (`Competition/SpawnStrategies` perhaps?) with only the main interface(s) in the `Competition` folder.
We may want to reorganise the `Control` and `Competition` folders at some point to separate out actual control stuff from local and remote competition stuff.